src/Microsoft.OpenApi.CSharpAnnotations.DocumentGeneration/SpecificationGenerationMessages.cs

```diff
@@ -46,6 +46,10 @@ public static class SpecificationGenerationMessages
         public const string NoOperationElementFoundToParse =
             "No valid operation elements containing tag \"<url>\" and \"<verb>\" have been found in the annotation xml.";
 
+        public const string NotSupportedInAttributeValue =
+            "Provided \"in\" attribute values: {0} for corresponding parameters: {1} is not supported. "
+            + "Supported values are: {2}.";
```

Comment:
Consider a variation that should be simpler to read:
Parameters "p1, p2" contain unsupported "in" attribute values "v1, v2". Supported values are: "v3, v4".

Code based on the current format argument order:
"Parameters \"{1}\" contain unsupported \"in\" attribute values \"{0}\". Supported values are: {2}."